src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,24 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li className="is-active" data-cy="Tab" key={tab.id}>
+              <a
+                href="#tab-1"
```

Comment:
The `href` attribute is hardcoded to `#tab-1`. This should be dynamically set to `#${tab.id}` to ensure each tab links correctly.